src/style.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-bar {
+  position: relative;
+  font-family: "Avenir", serif;
+  font-weight: 300;
+  color: #3d4e61;
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus {
+  border-radius: 3px;
+  font-weight: bold;
+  outline: 0;
+}
+
+.search-bar--size-big {
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  padding-left: 62px;
+  background-position: top 25px left 26px;
+}
+
+.search-bar--size-small {
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  padding-left: 33px;
+  background-position: top 14px left 12px;
+  background-size: 11px 11px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.